content/patterns/radio/examples/radio-activedescendant.html

```diff
@@ -212,8 +211,8 @@ <h2 id="rps_label">Role, Property, State, and Tabindex Attributes</h2>
               <td><code>li</code></td>
               <td>
                 <ul>
-                  <li>Identifies the element as an ARIA <code>radio</code> button.</li>
-                  <li>The accessible name is computed from the child text content of the element.</li>
+                  <li>要素を ARIA <code>radio</code> ボタンとして識別します。</li>
+                  <li>アクセシブルな名前は要素の子テキストコンテンツから計算されます。</li>
```

Comment:
computedは機械的に求まることを指しているので、算出くらいがいいでしょうか
```suggestion
                  <li>アクセシブルな名前は要素の子テキストコンテンツから算出されます。</li>
```